src/miniscript/types/mod.rs

```diff
@@ -377,10 +367,10 @@ pub trait Property: Sized {
         fragment: &AstElem<Pk, Pkh>,
         mut child: C,
     ) -> Result<Self, Error<Pk, Pkh>>
-    where
-        C: FnMut(usize) -> Option<Self>,
-        Pk: Clone,
-        Pkh: Clone,
+        where
+            C: FnMut(usize) -> Option<Self>,
+            Pk: Clone,
+            Pkh: Clone,
```

Comment:
More formatting changes that `cargo fmt` is going to undo.

Reply:
Somehow my editor does this weird stuff automatically. If I run `cargo fmt` it would change many other things which are not supposed to be a part of this PR. 
Or if you want, I can add a `cargo fmt` commit to the this PR? Once we have `cargo fmt` ed version of the codebase, I can run `cargo fmt` before raising a PR. 
From the next PR onwards I can run `cargo fmt` locally without

Reply:
No, we'll do another `cargo fmt` PR because there are probably 100+ style issues in this PR. I was just commenting here because this introduces new style issues without even changing any code.

My suggestion is to switch to emacs ;)